lib/v2/screens/profile_screens/profile/interactor/mappers/profile_values_state_mapper.dart

```diff
@@ -23,6 +23,6 @@ class ProfileValuesStateMapper extends StateMapper {
   }
 }
 
-extension ValueResult<T> on Result<T> {
+extension _ValueResult<T> on Result<T> {
   T? get valueOrNull => isValue ? asValue!.value : null;
 }
```

Comment:
This is not a global extension? I think Nick used it in a previous PR and actually he aked to me where it should go.

Reply:
nothing broke when i made it private. 
If we want to make it global we need to move it out of here. 

I changed it because my IDE was freaking out with name and could not figure out the correct import